frontend/src/app/pages/exercises/exercise/shared/exercise-map/shared/personnel-popup/personnel-popup.component.scss

```diff
@@ -0,0 +1,3 @@
+.popup-width {
+    max-width: 20vw;
+}
```

Comment:
From the style guide in the frontend readme:

> -   We are using [Bootstrap](https://getbootstrap.com) as a style framework.
> -   Where should I put my CSS? (Guideline)
>     1. Use Bootstrap classes whenever possible (look at the [utility classes](https://getbootstrap.com/docs/5.0/utilities/api/)).
>     2. If you can reuse the styling, add a class to the [style.scss](./src/styles.scss) (or the components SCSS if it's only used there).
>     3. If the styling is custom and only applicable to one specific element, use inline CSS.
>     4. If the inline styling is very big, complicated, or makes use of any SCSS or advanced CSS not applicable to inline styles, put it into the components SCSS file.
>     -   See also [Bootstraps best practices](https://getbootstrap.com/docs/5.0/extend/approach/#summary).

In this case, I would either make a `.popup-width` style in the `styles.scss` file and reuse it in all popups or write the CSS directly inline.